site/content/customers/_index.md

```diff
@@ -21,7 +21,8 @@ customers:
   logos:
     - icon: '/case-studies/kizen.png'
     - icon: '/case-studies/joyn.svg'
-    - icon: '/case-studies/solutia.png'
```

Comment:
Do we want to remove this one?

Reply:
Yes! We still have the icon, that logo list is the one we iterate for customers without case study.